lib/modules/versioning/ubuntu/index.ts

```diff
@@ -6,15 +6,81 @@ export const displayName = 'Ubuntu';
 export const urls = ['https://changelogs.ubuntu.com/meta-release'];
 export const supportsRanges = false;
 
+const distroInfo = `Ubuntu 4.10 "Warty Warthog"
```

Comment:
Please convert to static json (you can create a js file to tools for it) and safe to data folder.
Then our `generate-imports` tool should automatically add it to the generated files.
checkout nodejs schedule file usage for sample

https://github.com/renovatebot/renovate/blob/4f65b57225a42f9deb010f8dbfeb06eb94c0dd69/tools/generate-imports.mjs#L66

https://github.com/renovatebot/renovate/blob/dca3418bbdbf1024ac6807e591562b4042bc846d/lib/modules/versioning/node/schedule.ts#L1

Reply:
thanks, exactly what i was aiming at 😄